### PR TITLE
Added core-packages pages

### PR DESCRIPTION
**Pull request by nitin42**
Added pages for `babel-traverse` and `babel-generator`.

### CODE REVIEW

**Comment by nitin42** [comment]
#1021

File: docs/plugins/transform-react-constant-elements.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: docs
 title: React constant elements transformer
-description:
+description: Treat React JSX elements as value types and hoist them to the highest scope.
```
Comment (existentialism) [inline docs/plugins/transform-react-constant-elements.md]: Wasn't this (and the other react plugin descriptions) already done in https://github.com/babel/babel.github.io/pull/1188?

Might want to rebase against master.

File: docs/core-packages.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ package: babel-core
 * [Babel-helpers](babel-helpers/): Collection of helper functions used by Babel transforms
 * [Babel-code-frame](babel-code-frame/): Generate errors that contain a code frame that point to source locations
 * [Babylon](babylon/): Babylon is a JavaScript parser used in Babel
+* [Babel-traverse](traverse/): The Babel Traverse module maintains the overall tree state, and is responsible for 	replacing, removing, and adding nodes.
```
Comment (existentialism) [inline docs/core-packages.md]: Nit: double space between `for` and `replacing`

Comment (existentialism) [inline docs/core-packages.md]: Also, I've found it strange that we capitalize Babel in the kebab cased package name here... I'd prefer:

Babel Types or babel-types

Comment (hzoo) [inline docs/core-packages.md]: I would prefer `babel-types`

Comment (nitin42) [inline docs/core-packages.md]: Change all the core-packages name to Kebab case ?

Comment (hzoo) [inline docs/core-packages.md]: yes

File: docs/core-packages/traverse.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+layout: docs
+title: babel-traverse
+description: The babel-traverse module maintains the overall tree state, and is responsible for replacing, removing, and adding nodes.
+permalink: /docs/core-packages/traverse/
+package: babel-traverse
+---
+
+{% include package_readme.html %}
```
Comment (xtuc) [inline docs/core-packages/traverse.md]: There is no README for babel-traverse, the page is empty on the website.

I made this https://gist.github.com/xtuc/26931c424fb7148c000217de8e7fdf76 some time ago. We could publish this.

File: docs/core-packages/generator.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+layout: docs
+title: babel-generator
+description: Turns an AST into code.
+permalink: /docs/core-packages/generator/
+package: babel-generator
+---
+
+{% include package_readme.html %}
```
Comment (xtuc) [inline docs/core-packages/generator.md]: The options table is broken, I will look at it.

screenshot from 2017-03-16 08-38-16

Comment (xtuc) [inline docs/core-packages/generator.md]: Done in 30da5ad

File: docs/core-packages.md (hunk 2 above)
Comment (nitin42) [inline docs/core-packages.md]: done

File: docs/core-packages/traverse.md (hunk 3 above)
Comment (nitin42) [inline docs/core-packages/traverse.md]: I created a PR two days ago : https://github.com/babel/babel/pull/5465

**Pull request closed without merging**